src/Kafka/Callbacks.hs

```diff
@@ -39,7 +40,7 @@ logCallback callback k =
   let realCb _ = callback . toEnum
   in rdKafkaConfSetLogCb (getRdKafkaConf k) realCb
 
--- | Add a callback for stats.
+-- | Add a callback for stats. The passed bytestring contains an utf8 encoded JSON document and can e.g. be parsed using Data.Aeson.decodeStrict.
```

Comment:
Maybe we can also add a link to e.g. https://github.com/edenhill/librdkafka/blob/master/STATISTICS.md to help users understand what to expect?